src/chrome/content/rules/MERY.jp.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+
+	Problematic subdomains:
+
+		- assets	(akamai)
+		- bravo	(akamai)
+		- cashmere	(akamai)
+		- content	(akamai)
+		- galena	(akamai)
```

Comment:
I think it's just a problem when you redirect something like `images.foo.bar` to `eu1337.cdn.com/foo-images`, but as long as content live on their own domain it souldn't be a problem.